lib/openstates/api.rb

```diff
@@ -15,6 +15,11 @@ def bills(options = {})
         end
       end
 
+      if options[:id]
+        url << "#{options[:id]}/"
+        options.delete(:id)
+      end
```

Comment:
I had to test the api calls manually to get it working right. Their api is a little badly documented.